cfgov/v1/migrations/0052_auto_20160224_2201.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import wagtail.wagtailcore.fields
+import wagtail.wagtailcore.blocks
+import wagtail.wagtailimages.blocks
+
+
+def migrate_text_intro_blocks(apps, schema_editor):
+    SublandingPage = apps.get_model('v1.SublandingPage')
+
+    for page in SublandingPage.objects.all():
+        for i, child in enumerate(page.header.stream_data):
+            if child['type'] == 'text_introduction':
+                page.content.stream_data = [child] + page.content.stream_data
+                del page.header.stream_data[i]
```

Comment:
I believe `del child` will only delete the reference to the object in the list, not the actual object.